srcpkgs/opensnitch/patches/cc-check.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/daemon/netfilter/queue.go b/daemon/netfilter/queue.go
+index 3797486baf..4ee09d68b5 100644
+--- a/daemon/netfilter/queue.go
++++ b/daemon/netfilter/queue.go
+@@ -3,7 +3,7 @@ package netfilter
+ /*
+ #cgo pkg-config: libnetfilter_queue
+ #cgo CFLAGS: -Wall -I/usr/include
+-#cgo LDFLAGS: -L/usr/lib64/ -ldl
++#cgo LDFLAGS: -L/usr/lib64 -ldl
```

Comment:
Is this so our cross-cc wrapper catches it?

If yes, please document on top of the patch